src/Components/DetailPage/DetailsPage.js

```diff
@@ -16,30 +19,103 @@ export const DetailsPage = () => {
           </div>
 
           <div className="info activity-title">
-            
-            <h1 className="">information goes here</h1>
+            <h1 className="title">Activity Title</h1>
+            <h1 className="">{
+               props.randomActivity.song_title 
+            || props.randomActivity.title 
+            || props.randomActivity.podcast_title 
+            || props.randomActivity.podcast_name
+            || props.randomActivity.name
+            || "more info soon..."}</h1>
           </div>
 
           <div className="info activity-information">
             <div className="image-container">
-              <img src="https://images.unsplash.com/photo-1533618821901-9e69d5f5360e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60" alt=""/>
+              {props.randomActivity.image_poster 
+              || props.randomActivity.image_100 || 
+              props.randomActivity.image ? 
+              <img 
+              src={props.randomActivity.image_poster 
+                  || props.randomActivity.image_100 
+                  || props.randomActivity.image 
+                  || noImage
+                  } alt={
+                    props.randomActivity.description
+                  }/> :
+            <div className="image-container" >
+              <ReactPlayer
+                alt="movie trailer"
+                controls={true}
+                width={350}
+                height={250}
+                url={props.randomActivity.video}
+              />
+            </div>}
             </div>
-            <p className="content">Here is a lot of the content, just in case you were wondering</p>
+
+            <p className="content">{
+               props.randomActivity.podcast_name
+            || props.randomActivity.artist
+            || props.randomActivity.title 
+            || "more information comming up soon"
+            }</p>
+
+            
+            {props.randomActivity.episode_count && <p className="content">{props.randomActivity.episode_count } episodes</p>}
+            
+            {props.randomActivity.average_time && <h4 className="title">Average play time</h4>}
+            {props.randomActivity.average_time && <p className="content">{props.randomActivity.average_time}</p>}
+            
+            {props.randomActivity.materials && <h4 className="title">Materials</h4>}
+            {props.randomActivity.materials && <p className="content">{props.randomActivity.materials}</p>}
+
+            {props.randomActivity.imdb_rating && <h4 className="title">Imdb Raiting</h4>}
+            {props.randomActivity.imdb_rating && <p className="content">{props.randomActivity.imdb_rating || 0}</p>}
+
+            {props.randomActivity.content_rating&& <h4 className="title">Content</h4>}
```

Comment:
If there's time, we might find it would be useful to move this logic out of the render and into a function that gets invoked here in the render - might be cleaner but is certainly not a big priority today.